src/js/app.js

```diff
@@ -987,6 +991,32 @@ var getCryptocurrencies = function() {
     }
 };
 
+
+// got it from https://github.com/stefanheule/graphite
+var getPhoneBattery = function () {
+    if (!navigator.getBattery) {
+        var data = {
+            "KEY_PHONEBATTERY_LEVEL": 101,
```

Comment:
little nit on formatting on these 2 blocks to keep the same indentation

Reply:
Nor my text editor nor me have no idea about indentation here:(. I am completely unfamilliar with js, sorry. Can you point me what exactly wrong here?